src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -15,14 +15,21 @@
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    internal sealed class SpillSequenceSpiller : BoundTreeRewriterWithStackGuard
+    internal class SpillSequenceSpiller : BoundTreeRewriterWithStackGuard
     {
         private const BoundKind SpillSequenceBuilderKind = (BoundKind)byte.MaxValue;
 
         private readonly SyntheticBoundNodeFactory _F;
         private readonly PooledDictionary<LocalSymbol, LocalSymbol> _tempSubstitution;
         private readonly PooledDictionary<LocalSymbol, BoundComplexConditionalReceiver> _receiverSubstitution;
 
+        private SpillSequenceSpiller(SpillSequenceSpiller parent)
```

Comment:
>(SpillSequenceSpiller parent) [](http://example.com/codeflow?start=36&length=29)

Lets have explicit constructor declarations. Under Compilers we prefer longer, but more explicit code.